src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -89,4 +91,16 @@ private void addOverseasEntityResourceToTransaction(Transaction transaction,
         transaction.setResources(Collections.singletonMap(submissionUri, overseasEntityResource));
         transactionService.updateTransaction(transaction, passthroughTokenHeader);
     }
+
+    public Optional<OverseasEntitySubmissionDto> getOverseasEntitySubmission(String submissionId) {
+        var submission = overseasEntitySubmissionsRepository.findById(submissionId);
+        if (submission.isPresent()) {
+            ApiLogger.info(String.format("%s: Confirmation Statement Submission found. About to return", submission.get().getId()));
+
+            var json = overseasEntityDtoDaoMapper.daoToDto(submission.get());
```

Comment:
i'd rename json var to dto as it's not strictly json (string) but the dto object